.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ executors:
     environment:
       GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError"'
     docker:
-      - image: cimg/openjdk:15.0.1
+      - image: cimg/openjdk:16.0.0
```

Comment:
OpenJDK 16.0.1 is available already but I don't see a corresponding version for cimg/openjdk yet, unfortunately. We can follow-up with a patch version bump when that is available.

Reply:
It's available now. I'm updating the pull request